docs/usage/redux-logic-thunks.md

```diff
@@ -0,0 +1,443 @@
+---
+id: writing-logic-thunks
+title: Writing Logic with Thunks
+description: 'Usage > Redux Logic > Thunks: writing logic that interacts with the store'
+---
+
+:::tip What You'll Learn
+
+- What "thunks" are, and why they're used for writing Redux logic
+- How the thunk middleware works
+- Techniques for writing sync and async logic in thunks
+- Common thunk usage patterns
+
+:::
+
+## Thunk Overview
+
+### What is a "thunk"?
+
+The word "thunk" is a programming term that means ["a piece of code that does some delayed work"](https://en.wikipedia.org/wiki/Thunk). Rather than execute some logic _now_, we can write a function body or code that can be used to perform the work _later_.
+
+For Redux specifically, "thunks" are a pattern of writing logic that can interact with a Redux store, and require the [`redux-thunk` middleware](https://github.com/redujs/redux-thunk) to be added to the Redux store as part of its configuration.
```

Comment:
I think this should be split up into 2 sentences. Especially with RTK building thunks in, the second sentence about middleware seems like a minor distraction here